services/agent/src/simcore_service_agent/_main.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+from servicelib.logging_utils import config_all_loggers
+
+from ._app import Application
+from ._settings import ApplicationSettings
+from .info import info_exposer
+from .volumes_cleanup import backup_and_remove_volumes
+
+
+def setup_logger(settings: ApplicationSettings):
+    # SEE https://github.com/ITISFoundation/osparc-simcore/issues/3148
+    logging.basicConfig(level=settings.LOGLEVEL.value)  # NOSONAR
+    logging.root.setLevel(settings.LOGLEVEL.value)
+    config_all_loggers()
+
+
+def create_application() -> Application:
+    app = Application()
+
+    settings = ApplicationSettings.create_from_envs()
+    setup_logger(settings)
+
+    app.add_job(
+        backup_and_remove_volumes,
+        settings,
+        repeat_interval_s=settings.AGENT_VOLUMES_CLEANUP_INTERVAL_S,
+    )
+    app.add_job(info_exposer, app)
```

Comment:
so these 2 jobs are totally separated. if the first job stops working, is it guaranteed that the info_exposer will stop too?

Reply:
out of date, but it is guaranteed that all the jobs stop when the `task_monitor` is stopped.